### ISSUE TITLE
Downloading fails with large QA Tile files

### ISSUE THREAD

**Issue by Geoyi** [developmentseed/label-maker#35]
While `label-maker download` mbtiles for `united_states_of_america` casues  Invalid argument as follows:
```
File "/Users/xxx/xxxx/label-maker/label_maker/download.py", line 32, in download_mbtiles
    w.write(r.read())
OSError: [Errno 22] Invalid argument
```
after the mbtiles has showed be downloaded.

**Comment by drewbo** [comment]
@Geoyi do you know if you had available space on machine? Is there any else specific about your machine configuration that might make this fail? I wasn't able to reproduce

**Comment by Geoyi** [comment]
@drewbo, I actually have about 96G on my machine. If I change to another country with a different bounding box, it works, but it just did not work for `united_states_of_america` if you and Mark could create and fetch the data without the problem, I think we could close this issue up.

**Comment by drewbo** [comment]
Ah sorry, I actually had a failure in another window. I can reproduce. The file is likely too large after unzipping to read/write properly in the currently implementation.

**Comment by Ashot-KR** [comment]
same for `russia`

**Comment by Ashot-KR** [comment]
@Geoyi as temporary solution you may download mbtiles directly
`https://s3.amazonaws.com/mapbox/osm-qa-tiles-production/latest.country/united_states_of_america.mbtiles.gz`
it works for me

**Comment by Geoyi** [comment]
Thanks, @Ashot-KR, glad that works for your case.

**Comment by drewbo** [comment]
Closed via #47
